python/lsst/pipe/analysis/plotRhoStatistics.py

```diff
@@ -0,0 +1,347 @@
+import numpy as np
+from matplotlib import pyplot as plt
+import treecorr
+
+from lsst.daf.persistence import Butler
+import lsst.afw.table as afwTable
+from utils import E1, E2, E1Resids, E2Resids, E1ResidsHsmRegauss, E2ResidsHsmRegauss, PsfTraceSizeDiff
+
+def corr2(ra, dec, g1a, g2a, g1b, g2b, ra_units="degrees", dec_units="degrees", **treecorr_kwargs):
+    """ Function to compute auto-/cross-correlations between atmost two shear fields.
+
+        This is used to compute Rowe statistics, given the appropriate 'shear' fields.
+    """
+
+    xy = treecorr.GGCorrelation(**treecorr_kwargs)
+    catA = treecorr.Catalog(ra=ra, dec=dec, g1=g1a, g2=g2a, ra_units="degrees",
+                               dec_units="degrees")
+    if g1b is None or g2b is None:
+        # Calculate the auto-correlation
+        xy.process(catA)
+    else:
+        catB = treecorr.Catalog(ra=ra, dec=dec, g1=g1b, g2=g2b, ra_units="degrees",
+                               dec_units="degrees")
+        # Calculate the cross-correlation
+        xy.process(catA, catB)
+
+    return xy
+
+# RFC: Should we move this to utils.py?
+def calculateRhoStatistics(ra, dec, e1, e2, e1Res, e2Res, SizeRes, ra_units="degrees", dec_units="degrees",
+        **kwargs):
```

Comment:
Add proper docs (as Lauren said don't use the rest of pipe_analysis as an example - make it better). Also I think the indent would look better aligned. Have you PEP8ed?

Reply:
This file is probably not PEP8ed, but I will do it (again?) for the rest. And yes, adding docstrings as @laurenam suggested.